hack/manifests/backstage/postgres.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
my understanding is that most of the time, db should be statefulset for persist storage.  in the future our backend need to use an external db too for production(h2 should only be for local testing)

Reply:
yes let's keep it in deploy